byte ADD=0
byte REMOVE=1
byte TOGGLE_IS_ACTIVE=2
byte UPDATE_INDEX = 3
byte operation

rule_monitor/TriggerType type
string name
string function
string description
bool is_active
int8 new_index